### ISSUE TITLE
2.3.0-rc1 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1590]
## Bug fixes
- #1571
- #1563
- #1588 
- #1586
- #1582 
- #1578

## Drupal contrib module updates
- #1568

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick 19af03fd66a8f3b575cfa60ad5623ce8a9b2f5fc
git cherry-pick 353915076cd83fd9e5dca3379970f072717e092d
git cherry-pick 0bc0136c85999d23b981bddc9c85d512de4b237a
git cherry-pick e6a8af3a17a3b2138eefabd9ed2ffcee403f75dc
git cherry-pick 2c67504b2c7a2633e5cb022084c8cc9f0310a74c
git cherry-pick 1f22f25e093bd47ac4a6536161eb4e47a394cbd2
git cherry-pick 4c21f2ff98c54fbce480153b4096e08a47ad71e1
```

## Release notes snippet

`git log 2.3.0-beta1..HEAD --pretty='format:- %s'`

```
- #1340 Allow import of related permissions in addition to provided permissions. (#1578)
- Update drupal/environment_indicator requirement from 4.0.5 to 4.0.6 (#1568)
- Closes #1581: Remove non-existent permissions from Quickstart roles. (#1582)
- Fixes #1583 Downgrading menu_block module from 1.8.0 to 1.7.0 (#1586)
- Closes #1584: Fix sidebar nav top margin calculation (#1588)
- Closes #1574 Update text in description of crop type `az_full_width_bg_large` (#1563)
- Closes #1570: Ensure better_exposed_filters module is installed. (#1571)
```

**Comment by joeparsons** [comment]
Released
